bin/setup_configuration.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# setup initial configuration using environment variables
+# Run this script from the root of the repository
+
+set -e
+
+if [[ "${RUN_SETUP_CONFIG,,}" =~ ^(true|1|yes)$ ]]; then
+    # wait for required services
+    /wait_for_db.sh
```

Comment:
```suggestion
    ./wait_for_db.sh
```
these scripts need to be debugged outside of docker context too, so you can't assume that they're available in the root of the system.